lib/vmpooler/vsphere_helper.rb

```diff
@@ -196,31 +196,62 @@ def find_folder(foldername)
       base
     end
 
+    def get_host_utilization(host, model=nil, limit=90)
```

Comment:
I added some comments to try and provide some context for what these means. Let me know if you think further comments are necessary, or would be helpful.